### PR TITLE
Add support for Pocketbook Color Lux

### PR DESCRIPTION
**Pull request by thomasrebele**
This solves issue #4193.

### CODE REVIEW

File: frontend/device/pocketbook/device.lua [hunk 1]
```diff
@@ -203,6 +209,18 @@ local PocketBook740 = PocketBook:new{
...
 
+-- PocketBook Color Lux
+local PocketBookColorLux = PocketBook:new{
+    isTouchDevice = yes,
+    hasKeys = yes,
+    hasFrontlight = yes,
+    hasColorScreen = yes,
+    has3BytesWideFrameBuffer = yes,
+    isAlwaysPortrait = no,
+    display_dpi = 100,
```
Comment (NiLuJe) [inline frontend/device/pocketbook/device.lua]: You can probably leave it out, fb:getDPI will fall back to 160, which should be roughly okay (it's was we do on other 600x800 devices) ;).

File: frontend/device/generic/device.lua [hunk 2]
```diff
@@ -46,6 +46,9 @@ local Device = {
     isAlwaysPortrait = no,
     -- needs full screen refresh when resumed from screensaver?
     needsScreenRefreshAfterResume = yes,
+    -- framebuffer reports 8bit per pixel, but is actually 24bit per pixel
```
Comment (NiLuJe) [inline frontend/device/generic/device.lua]: I'd add a link to your original issue in the comment, in case that confuses someone down the road ;).

**Comment by NiLuJe** [comment]
Besides those two minor comments, LGTM ;).

File: frontend/device/generic/device.lua (hunk 2 above)
Comment (NiLuJe) [reply frontend/device/generic/device.lua]: Possibly even saying right then and there that it's a Triton/color eInk screen

**Comment by Frenzie** [comment]
Pinging @thomasrebele about the small comments.

**Comment by Frenzie** [comment]
@thomasrebele Thanks! Could you please add a base bump to this PR to wrap it all up?

**Comment by thomasrebele** [comment]
I hope I bumped it correctly. Thanks again for all the help. Koreader is a great project.

**Comment by Frenzie** [comment]
Darn, it seems to have caused some kind of conflict. You could try to fix it with rebasing, e.g., using the guide from http://koreader.rocks/doc/topics/Collaborating.html or I can also take care of it, albeit not right at this very moment. (Hence why I asked you to bump base. :-) )

**Comment by thomasrebele** [comment]
thanks for fixing it!

**Pull request merged**